scripts/archlinux/solidity-git/PKGBUILD

```diff
@@ -0,0 +1,68 @@
+# Maintainer: Afri 5chdn <aur@cach.co>
+# Available from AUR: https://aur.archlinux.org/packages/solidity-git/
+
+pkgname=solidity-git
+pkgver=0.4.3.git.20161020.9d30450
+pkgrel=1
+pkgdesc="The Solidity Contract-Oriented Programming Language (Including solc, lllc; from latest unstable git version)"
+arch=('i686' 'x86_64')
+depends=(
+  'boost'
+  'boost-libs'
+  'crypto++'
```

Comment:
I don't think solidity depends on crypto++ anymore. I don't have it installed in my laptop and I can still build solidity.